README.md

```diff
@@ -116,6 +117,14 @@ Return the `.default` export of a module at the top-level.
 
 Custom alias map used to resolve ids.
 
+### `jsx`
+
+- Type: Boolean
```

Comment:
(we can document value can be a stringified JSON object too)

Reply:
It has been updated.